test/test_images/bloatingcow/bloatingcow.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+func bloat(mb int) {
+	if mb == 0 {
+		return
+	}
+	fmt.Printf("Bloat %v Mb of memory.\n", mb)
+
+	b := make([]byte, mb*1024*1024)
+	b[0] = 1
+	b[len(b)-1] = 1
```

Comment:
Perhaps initialize the whole thing for good measure?

Reply:
I'm worried about the allocation happening virtually and then getting filled in through page faults as the pages are actually consumed.  I doubt first and last is good enough for all sizes.